examples/protocols/sntp/main/sntp_example_main.c

```diff
@@ -157,10 +166,43 @@ static void initialize_sntp(void)
 {
     ESP_LOGI(TAG, "Initializing SNTP");
     sntp_setoperatingmode(SNTP_OPMODE_POLL);
+
+/*
+ * If 'NTP over DHCP' is enabled, we set dynamic pool address
+ * as a 'secondary' server. It will act as a fallback server in case that address
+ * provided via NTP over DHCP is not accessible
+ */
+#if defined LWIP_DHCP_GET_NTP_SRV && (SNTP_MAX_SERVERS > 1)
+    sntp_setservername(1, "pool.ntp.org");
+
+#if LWIP_IPV6                   // statically assigned IPv6 address is also possible
+    ip_addr_t ip6;
+    if (ipaddr_aton("2a01:3f7::1", &ip6))       // ipv6 ntp source "ntp.netnod.se"
+        sntp_setserver(2, &ip6);
```

Comment:
Formatting (see the contribution guide)
```suggestion
    if (ipaddr_aton("2a01:3f7::1", &ip6)) {
         // ipv6 ntp source "ntp.netnod.se"
        sntp_setserver(2, &ip6);
    }
```